### PR TITLE
MODLD-352: Story[Marc2BF]: Convert marc field 008, chars 35-37 to "la…

### PR DESCRIPTION
**Pull request by askhat-abishev**
…nguage" edge of Work

### CODE REVIEW

File: src/main/resources/fingerprint-rules.yml [hunk 1]
```diff
@@ -30,6 +28,8 @@ rules:
         label: true
       - predicate: SUBJECT
         label: true
+      - predicate: LANGUAGE
+        label: true
```
Comment (pkjacob) [inline src/main/resources/fingerprint-rules.yml:32]: Let us use code instead of label.

Comment (askhat-abishev) [reply src/main/resources/fingerprint-rules.yml]: Changed

File: src/main/resources/fingerprint-rules.yml [hunk 2]
```diff
@@ -30,6 +28,9 @@ rules:
         label: true
       - predicate: SUBJECT
         label: true
+      - predicate: LANGUAGE
+        edgeProperties:
+          CODE: http://bibfra.me/vocab/marc/code
```
Comment (pkjacob) [inline src/main/resources/fingerprint-rules.yml:33]: Let us use the URI "http://bibfra.me/vocab/lite/language" for language code in fingerprint. "http://bibfra.me/vocab/marc/code" will not indicate that the code is for language.

`CODE: http://bibfra.me/vocab/lite/language`

**Review by PBobylev — APPROVED** [review]

File: src/main/resources/fingerprint-rules.yml (hunk 2 above)
Comment (askhat-abishev) [reply src/main/resources/fingerprint-rules.yml]: Done

**Pull request merged**